supervisor/shared/web_workflow/web_workflow.c

```diff
@@ -60,13 +60,9 @@
 #include "shared-bindings/socketpool/Socket.h"
 #include "shared-bindings/socketpool/SocketPool.h"
 
-#if CIRCUITPY_WIFI
 #include "shared-bindings/wifi/__init__.h"
-#endif
 
-#if CIRCUITPY_OS_GETENV
 #include "shared-module/os/__init__.h"
-#endif
```

Comment:
This makes sense, because `supervisor/supervisor.mk` only compiles this file if `CIRCUITPY_WEB_WORKFLOW = 1`